cypress/automated-tests/youth-pass/successful-youth-pass-submission.spec.js

```diff
@@ -0,0 +1,65 @@
+const faker = require('faker');
+const constants = require('../../common/youth-pass-constants');
+
+describe('Youth Pass Successful Submission', () => {
+    it('Submits a Youth Pass Application', () => {
+        const youthPassUrl = constants.getYouthPassConstants().youthPassUrl;
+        const applicantFirstName = faker.name.firstName();
+        const applicantLastName = faker.name.lastName();
+        const applicantEmailAddress = `Automation_Testing_${faker.datatype.number()}@example.com`;
```

Comment:
While faker does provide email addresses, they use real domains. This avoids that potential pitfall and also allows the RF team to quickly see which submissions come from automation.

Reply:
Yeah, I appreciate you using `example.com`, I'm surprised the library wouldn't default to that.